examples/tests/sdk_tests/std/string.w

```diff
@@ -40,21 +40,148 @@ test "length" {
 // at()
 
 assert("boom".at(2) == "o");
-//TODO: assertThrows("ERROR", () => { "hey".at(-1); });
+// Negative integers count back from the last string character.
```

Comment:
Is this documented in the api docs?

Reply:
It is documented in JS. Do you think we should document this? this is the expected behavior in JS.